easybuild/easyconfigs/x/xtb/xtb-6.6.1-foss-2022b.eb

```diff
@@ -0,0 +1,51 @@
+# Updated to 6.6.1, dependencies added as modules
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.6.1'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = """ xtb - An extended tight-binding semi-empirical program package. """
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['df9a9fbcbf685a94ba6e1a97a6fe6a8530227ea380a1507cb758e72907542dfe']
+
+builddependencies = [
+    ('Meson', '0.64.0'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '1.9.3'),
+    ('git', '2.38.1', '-nodocs'),
+]
+
+dependencies = [
+    ('tblite', '0.3.0'),
```

Comment:
Needs an upxdate to tblite 0.4.0

Reply:
`tblite` is now broken, probably due to some of the requested changes of the modules. The problem seems to be that different subprograms are using different versions of `TOML-Fortran` and as they need to be dependencies, they will be loaded. See also issue  [1091)](https://github.com/grimme-lab/xtb/issues/1091). I am facing similar issues with version `6.7.1`